operations/observability/mixins/meta/dashboards/components/server.json

```diff
@@ -678,7 +678,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(server_websocket_connection_count) by (cluster, clientType)",
+          "expr": "sum(server_websocket_connection_count{cluster=~\"$cluster\", pod=~\"$pod\"}) by (cluster, clientType)",
```

Comment:
Would it work if we don't add `pod` to the dashboard settings variable?

Reply:
You mean if we removed the variable `pod` from the dashboard entirely? Yes.

But this is the "Component / server" dashboard where I'd like to offer as much options for drill-down as possible. It was just that this graph did not respect those filters until now.

Reply:
Sorry I missed that it's in the server file! Looks good then.